var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -89,6 +89,7 @@ class PyAwkward(PythonPackage):
     depends_on("rapidjson")
     depends_on("cmake@3.13:", type="build")
     depends_on("py-wheel@0.36.0:", when="@:1.7.0", type="build")
+    depends_on("py-importlib-metadata@4.13.0:", when="@2.6.6:", type="run")
```

Comment:
The linked `pyproject.toml` constrains the dependency also on the version of python being used.  Should the type be `("build", "run")?

```
    "importlib_metadata>=4.13.0;python_version < \"3.12\"",
```

Reply:
It also lists a dependency on ``py-fsspec`:

```
    "fsspec>=2022.11.0"
```

Reply:
This dependency was also there for [2.6.5](https://github.com/scikit-hep/awkward/blob/v2.6.5/pyproject.toml#L44) and probably earlier.